addons/mail/static/tests/qunit_suite_tests/components/link_preview_tests.js

```diff
@@ -0,0 +1,415 @@
+            },
+        },
+    });
+    await openDiscuss();
+
+    assert.containsOnce(
+        document.body,
+        '.o_LinkPreviewCardView',
+        "should have link preview in DOM"
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_LinkPreviewCardView_title',
+        "Should display the link preview title"
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_LinkPreviewCardView_description',
+        "Link preview should show the link description"
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_LinkPreviewCardView_image',
+        "Should display an image inside the link preview card"
+    );
+});
+
+QUnit.test('Link preview video layout', async function (assert) {
+    assert.expect(4);
+
+    const pyEnv = await startServer();
+    const linkPreviewId = pyEnv['mail.link.preview'].create(link_preview_video_payload);
+    const mailChannelId = pyEnv['mail.channel'].create({});
+    pyEnv['mail.message'].create({
+        body: 'not empty',
+        link_preview_ids: [linkPreviewId],
+        message_type: 'comment',
+        model: "mail.channel",
+        res_id: mailChannelId,
+    });
+    const { openDiscuss } = await start({
+        discuss: {
+            params: {
+                default_active_id: `mail.channel_${mailChannelId}`,
+            },
+        },
+    });
+    await openDiscuss();
+
+    assert.containsOnce(
+        document.body,
+        '.o_LinkPreviewVideoView',
+        "should have link preview video in DOM"
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_LinkPreviewVideoView_title',
+        "Should display the link preview title"
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_LinkPreviewVideoView_description',
+        "Link preview should show the link description"
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_linkPreviewVideo_overlay',
+        "Should display overlay inside the link preview video image"
+    );
+});
+
+QUnit.test('Link preview image layout', async function (assert) {
+    assert.expect(1);
+
+    const pyEnv = await startServer();
+    const linkPreviewId = pyEnv['mail.link.preview'].create(link_preview_image_payload);
+    const mailChannelId = pyEnv['mail.channel'].create({});
+    pyEnv['mail.message'].create({
+        body: 'not empty',
+        link_preview_ids: [linkPreviewId],
+        message_type: 'comment',
+        model: "mail.channel",
+        res_id: mailChannelId,
+    });
+    const { openDiscuss } = await start({
+        discuss: {
+            params: {
+                default_active_id: `mail.channel_${mailChannelId}`,
+            },
+        },
+    });
+    await openDiscuss();
+
+    assert.containsOnce(
+        document.body,
+        '.o_LinkPreviewImageView',
+        "should have link preview image"
+    );
+});
+
+QUnit.test('Remove link preview Gif', async function (assert) {
```

Comment:
"gif" no longer a specific case

Reply:
It's still a specific case because it work with og_mimetype instead of image_mimetype